### PR TITLE
MESH-1800/ changed multiSig.MultiSigSigners map to bool

### PR DESCRIPTION
**Pull request by JMoore96**
### modified logic

- modified MultiSigSigners map

### CODE REVIEW

**Review by Neopallium — CHANGES_REQUESTED** [review]
Needs a storage migration.

**Comment by adamdossa** [comment]
Ideally we should use `get` instead of `contains_key` in e.g.:
`!<MultiSigSigners<T>>::contains_key(&multisig, &signer),`
to check that the key maps to false (rather than just being absent).

File: pallets/multisig/src/lib.rs [hunk 1]
```diff
@@ -1173,4 +1192,18 @@ mod migration {
...
     }
+
+    pub fn migrate_v2<T: Config>() {
+        sp_runtime::runtime_logger::RuntimeLogger::init();
+
+        log::info!(" >>> Updating MultiSig storage. Migrating MultiSigSigners..");
+        let total_ms_signers =
+            v2::MultiSigSigners::<T>::drain().fold(0usize, |total_ms_signers, (signer, ms, _)| {
+                // Migrate MultiSigSigner.
+                <MultiSigSigners<T>>::insert(&signer, &ms, true);
```
Comment (Neopallium) [inline pallets/multisig/src/lib.rs]: The parameter names are backwards.  The `ms` is the first key in the double map and `signer` is the second key.

**Review by Neopallium — CHANGES_REQUESTED** [review]
One minor change and it is ready.

**Review by Neopallium — APPROVED** [review]

**Review by adamdossa — APPROVED** [review]

**Pull request merged**